HelpSource/Tutorials/QT/create_simple_instrument_view.schelp

```diff
@@ -0,0 +1,548 @@
+title:: QT : Interfacing an instrument with a View
+summary:: Creating a view that interacts with a Synth, introduction to Layouts
+categories:: Tutorials>QT
+
+section:: What is this tutorial about?
+
+This is a tutorial to get started using SuperCollider's QT implementation to strong::create views and make them interact with a Synth::.
+
+It covers those topics :
+
+list::
+## Create a keyboard from a list of midi notes
+## Create a volume slider, and a duration slider
+## Use the layout system so widgets are displayed following a grid
+::
+
+section:: Setup
+
+
+You will need to strong::boot the Server::, to strong::load the SynthDef:: we will be using in this tutorial. It is a simple sine wave with a percussive enveloppe. We can control its frequency, its volume and its duration.
+
+code::
+(
+SynthDef(\instrView, { |out = 0, amp = 0.25, freq = 440, dur = 1.0|
+	var snd, env;
+
+	env = EnvGen.kr(Env.perc(dur * 0.01, dur * 0.99), doneAction: Done.freeSelf);
+	snd = SinOsc.ar(freq, mul: amp);
+	snd = snd * env;
+	snd = snd!2;
+
+	Out.ar(out, snd);
+}).add;
+)
+::
+
+You can try the synth using :
+
+code::
+Synth(\instrView)
+::
+
+We'll start with the following code :
```

Comment:
Maybe it is good to spoil that this will not create any sound yet? At least I tried clicking the button in expectation of a sound :)
```suggestion
We'll start with the following code which is not yet connected to our synth and will therefore not yield any sound yet:
```

Reply:
Manually fixed.